docs/faq.rst

```diff
@@ -65,6 +72,18 @@ Can the mitigation algorithms in Fairlearn make my model fair?
     However, to assess whether the new model is fairer, it is important to consider
     not only the fairness metrics, but also the societal and technical context in which
     the model is applied.
+
+I've got improved data, trained and mitigated new models, checked all the metrics... am I done?
+    The ML lifecycle doesn't end when a model is deployed.
```

Comment:
I would expand a little on the statement in the beginning about metrics. Metrics don't tell the whole story. Fairness is sociotechnical. So this may not even have been sufficient at deployment time. I do like your points about data drift etc., of course.

Reply:
Good point - even though I wrote the question, I didn't notice that it implicitly included a deploy/no-deploy choice. I've added a note to highlight this.